<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1445</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="IPEdit">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>80</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortEdit">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>130</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectBtn">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>120</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>80</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>130</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageEdit">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>230</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>230</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textShow">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>290</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>850</width>
     <height>850</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>chessboard.jpg</normaloff>chessboard.jpg</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
